rclpy/src/rclpy/_rclpy.c

```diff
@@ -3557,19 +3557,19 @@ rclpy_get_rmw_qos_profile(PyObject * Py_UNUSED(self), PyObject * args)
   }
   PyObject * pyqos_profile = NULL;
   if (0 == strcmp(pyrmw_profile, "qos_profile_sensor_data")) {
-    pyqos_profile = rclpy_common_convert_to_py_qos_policy(&rmw_qos_profile_sensor_data);
```

Comment:
If `rclpy_common_convert_to_py_policy` is not used anywhere else, I think we can remove it's definition in favor of `rclpy_common_convert_to_qos_dict`.